### DISCUSSION TITLE
为什么要在 mathjax 的 styl 文件里面设定 mjx-container 的 padding？

### DISCUSSION THREAD

**Discussion by hankeke303** [volantis-x/hexo-theme-volantis#891] [General]
如题，指的是 `/blog/themes/volantis/source/css/_style/_plugins/mathjax.styl` 这个文件：
```styl
.article
  mjx-container
    font-family: $fontfamily-code
    padding: $gap $gap * 0.5
    border-radius: $border-codeblock
    min-width: 0 !important
  mjx-container[jax="CHTML"][display="true"], .has-jax
    overflow: auto hidden
  mjx-container + br
    display: none
```
中的 `padding: $gap $gap * 0.5` 这一行。

这样会造成在使用 mathjax 的时候，公式两侧会有一个字母长度的空白，而我习惯于手动在 tex 公式两侧加空格，这样我自己的空格和由这里的 `padding` 带来的空格连在一起就会显得空白过长，这样似乎不太好看。

<img width="802" alt="image" src="https://github.com/volantis-x/hexo-theme-volantis/assets/28253614/b6146e2a-0666-41d4-80ab-671fb95af020">

我现在是使用手动注入外部文件修改了这个样式，但是我有点好奇为什么要这样设计？

**Comment by Tikas** [comment]
前后增加一个字母的空格，是一个好习惯，这有利于阅读体验的提高，因为很多人并不会自己手动增加空格。

当然，对于您有着这种良好习惯的来说，可以为这个功能配置一个开关，让用户来自己决定要不要自动两边增加。

非常不错的提议，如果是图片里的效果，我个人的感观是很舒适的，这能让我很直观知晓这是公式上的相关。
